src/main/java/ca/airspeed/FreshBooksLogoutController.java

```diff
@@ -24,12 +26,13 @@
 public class FreshBooksLogoutController {
 
     private final OauthClientConfiguration clientConfiguration;
-    private final OauthConfiguration oauthConfiguration;
     private final FreshBooksApiClient freshBooksApiClient;
+    private final LogoutHandler logoutHandler;
 
     @Get("/oauth/logout")
```

Comment:
You may want to this path to `/oauth/logout/freshbook`. That will be the convention used for OpenID Connect authorisation servers which expose an end session endpoint.

Reply:
Will do.